.github/workflows/lint.yml

```diff
@@ -1,7 +1,7 @@
 name: Lint Code Base
 
 on:
-  push:
+  pull_request:
```

Comment:
Does this mean it only happens once when opening the PR? What about pushing a commit to a branch with a PR open, would that rerun checks?

Reply:
It is hard to tell that from GH actions docs, but it will queue action on any push to opened PR branch

I manually checked this behavior in separate repository: https://github.com/ewgenius/test-repository/pull/1